simtools/applications/plot_simtel_histograms.py

```diff
@@ -86,14 +86,15 @@ def main():
             raise TypeError
 
     # Building SimtelHistograms
+    logger.info(f"Histograms will be produced for {histogram_files}.")
```

Comment:
Tiny comment, but maybe we have consistent language. My suggestion is to use the same as in line 97: `Creating pdf file`; `Creating histograms for {histogram_files}`

(Avoiding philosophical discussions about the differences between `creating` and `producing` :-) )